### ISSUE TITLE
Sensors change to unknown state intermittently

### ISSUE THREAD

**Issue by krazos** [bretterer/home-assistant-rivian#27]
Most if not all of my `sensor` and `binary_sensor` entities change to `unknown` state many times throughout the day.  Those blips generally last about 5 seconds before the entities return to their true state:

![image](https://user-images.githubusercontent.com/18428022/200068462-2a8ab1de-bea2-4969-b1d3-aa508c5a6838.png)

I am not seeing the same behavior from the `device_tracker` entity.

I have not had a chance to dig into the code to see what may be going on, but I will try to take a look when I can.

**Comment by bretterer** [comment]
Thanks @krazos.  I believe this to be an issue related to the data we are getting from Rivian.  We are looking into this and possibly swapping over to GraphQL fully which should resolved this.

**Comment by krazos** [comment]
Interestingly enough, these momentary blips have been much improved since yesterday afternoon.  I am probably jinxing myself, but I don't think my sensors have flipped to `unknown` since 2:34 PM EST yesterday.

**Comment by p3ck** [comment]
Agreed.  Same for me, sensors are no longer showing the unknown blips.

Good work!

**Comment by tmack8001** [comment]
Could be a backend fix from Rivian, not sure if we changed anything that would improve that 🤔

**Comment by jrgutier** [comment]
I fixed this in #30, along with some new sensors

**Comment by bretterer** [comment]
This appears to be resolved now.  If there is still an issue, please open a new issue referencing this one
